src/views/Issues.vue

```diff
@@ -77,7 +80,8 @@ export default {
     }),
     async loadIssuesWithFilters() {
       this.loading = true
-      this.issues = await this.loadIssues(this.filters)
+      await this.loadIssues(this.filters)
+      this.issues = this.sortBy('priority', true)
```

Comment:
Vuex state could be used directly in template instead using a "temporary property" like `this.issues`